### PR TITLE
Remove old CMP

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Removes the old CMP, as it is currently dead code, which is already tree shaken away in the bundles.
All CMP actions are handled by https://github.com/guardian/consent-management-platform/.

Somewhat follows from #22909.

## Does this change need to be reproduced in dotcom-rendering ?

- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## What is the value of this and can you measure success?

Smaller codebase.

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [X] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [X] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [X] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/reader-revenue-dev-utils.js [hunk 1]
```diff
@@ -20,7 +20,6 @@ import {
 import { setGeolocation, getSync as geolocationGetSync } from 'lib/geolocation';
 import config from 'lib/config';
 import { clearParticipations } from 'common/modules/experiments/ab-local-storage';
-import { COOKIE_NAME as CONSENT_COOKIE_NAME } from 'commercial/modules/cmp/cmp-env';
```
Comment (mxdvl) [inline static/src/javascripts/projects/common/modules/commercial/reader-revenue-dev-utils.js:23]: I believe this is only related to the old CMP.

cc. @guardian/reader-revenue

Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/commercial/reader-revenue-dev-utils.js:23]: 👍

File: tools/__tasks__/compile/data/index.js [hunk 2]
```diff
@@ -1,9 +1,4 @@
 module.exports = {
     description: 'Clean download and build data assets',
-    task: [
-        require('./clean'),
-        require('./download'),
-        require('./aib_cmp'),
```
Comment (mxdvl) [inline tools/__tasks__/compile/data/index.js:6]: This is the one removed

**Review by sndrs — APPROVED** [review]
oh wow great spot!

**Comment by ioanna0** [comment]
Nice 👍 I have also spotted few endpoints we will need to get rid of:

https://github.com/guardian/frontend/blob/33b09199871ce9716f972f129f01686b9bf9f5f5/commercial/conf/routes#L39
https://github.com/guardian/frontend/blob/33b09199871ce9716f972f129f01686b9bf9f5f5/commercial/conf/routes#L40

but probably better to remove on a subsequent PR & card it up

**Pull request merged**